curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4ad.md

```diff
@@ -0,0 +1,67 @@
+---
+id: 66c8ba975ee7230e29f6c4ad
+title: Step 6
+challengeType: 1
+dashedName: step-6
+---
+
+# --description--
+
+To check which loan a user is qualified for based on the `annualIncome` and `creditScore` they entered, you have to use `if/else if` statement, AKA conditonals.
+
+Here's the basic syntax of an `if/else if` statement:
+
+```js
+if (condition1) {
+  // Do something if condition1 is true
+} else if (condition2) {
+  // Do something else if condition1 is false and condition2 is true
+} else {
+  // Do something else if both condition1 and condition2 are false
+}
+```
+
+You can also combine conditions inside a single `if` statement with the `AND` and `OR` operator like this:
+
+```js
+if (condition1 && condition2) {
+  // Do something if both conditions are true
+}
+
+if (condition1 || condition2) {
+  // Do something if either condition1 or condition2 is true
+}
+```
```

Comment:
I don't know if we will need this example either.
because there will be a lecture video and labs before this where they will learn about logical operators